src/__tests__/components/Modal/Modal.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { configure, mount } from 'enzyme';
+import toJSON from 'enzyme-to-json';
+import Adapter from 'enzyme-adapter-react-15';
+
+import Modal from '../../../components/Modal/Modal';
+
+configure({ adapter: new Adapter() });
```

Comment:
This `configure` statement should be inside of a **setup** file like this one:
https://github.com/hellofresh/my-deliveries-fragment/blob/master/__tests__/setup.js

Also, take a look at this file:

https://github.com/hellofresh/my-deliveries-fragment/blob/master/.jestrc#L2-L5

It contains the instructions to provide a **setup** file.